src/Gui/SettingsScreen/ColorCallback.java

```diff
@@ -0,0 +1,7 @@
+package Gui.SettingsScreen;
+
+import javafx.scene.paint.Color;
+
+public interface ColorCallback {
+    void onColourChange(Color color);
```

Comment:
Ik denk dat het het handigst is als we de Amerikaanse spelling aanhouden hier? Gezien we vanuit JavaFX al Color gebruiken, wel zo consistent? Wat denk jij?

Reply:
Geen probleem moet lukken!